regulations/templates/regulations/preamble-partial.html

```diff
@@ -3,10 +3,9 @@
 <section id="{{full_id}}" class="reg-section" data-page-type="preamble-section" data-doc-id="{{doc_number}}">
   <ul id="preamble-read">
     {% render_nested template=sub_template context=sub_context %}
+    {% include "regulations/footnotes.html" %}
```

Comment:
That'd work if the indent isn't too weird. If it is, we can probably wrap the `ul` in a `div#preamble-read` and place the footnotes as a sibling of the `ul`